app/services/rebate_forms_service.rb

```diff
@@ -5,6 +5,28 @@ def initialize(rebate_form_attributes)
     @rebate_form_attributes = rebate_form_attributes
   end
 
+  def generate_qr(rebate_form)
+    # create a new token that only allows you to fetch a restricted set of details for this application only
+    # and use the same token to submit the signatures back
+    # this token is valid for 30 minutes
+
+    payload = {
+      rebate_form_id: rebate_form.id,
+      exp: Time.now.to_i + (1000 * 60),
```

Comment:
🚨 change this back to 30 minutes

I strongly recommend making the duration a variable, maybe a constant defined up the top of the class. It helps future readers to spot magic numbers even if you're not planning to change the duration at this time.

Reply:
Anyone against an ENV setting? Can't have enough of those.